content/techniques/configuration.md

```diff
@@ -565,3 +565,29 @@ You can then use it as usual, by calling the `get` method with the configuration
 ```typescript
 const port = configService.get('PORT');
 ```
+
+#### Edge Cases
+
+If a decorator depends on an environment variable's existance, `dotenv` can be used directly in the `main.ts` file. This is particularly useful for the `@Cron` decorator.
+
+Preload the environment variables:
+
+```typescript
+@@filename(main)
+import dotenv from 'dotenv'
+dotenv.config()
+import {AppModule} from ./app.module
+```
```

Comment:
Can we make some sort of mention here that the import and `config()` call **have** to happen before importing `./app.module`? If it isn't, then `process.env` will not be defined at the time of the decorator function's execution